packages/engine-server/src/markdown/remark/dendronPub.ts

```diff
@@ -159,7 +162,7 @@ function plugin(this: Unified.Processor, opts?: PluginOpts): Transformer {
               value: alias,
             },
           ],
-        };
+        } as any /* type checker otherwise complains about hName */;
```

Comment:
instead of casting to `any`, can we create a custom type `RehypeLinkData` and add it [here](https://github.com/dendronhq/dendron/blob/dev/packages/engine-server/src/markdown/types.ts#L80:L80)?